www/janf04/semestralka1/hraci.html

```diff
@@ -96,28 +102,41 @@ <h3>Daniel Venhoda</h3>
     <div class="wrapper">
         <h2>Útok</h2>
         <div class="container">
-            <div class="float-box">
-                <h3>Tuan Anh Nguyen</h3>
-                <p>Výška: 170 cm</p>
-                <p>Číslo dresu: 7</p>
-                <p>Přezdívka: Marty, CR7</p>
-            </div>
-            <div class="float-box">
-                <h3>Michal Špinar</h3>
-                <p>Výška: 180 cm</p>
-                <p>Číslo dresu: 6</p>
-                <p>Přezdívka: Sněží</p>
+            <div class="attack">
+                <div class="float-box">
+                    <h3>Tuan Anh Nguyen</h3>
+                    <p>Výška: 170 cm</p>
+                    <p>Číslo dresu: 7</p>
+                    <p>Přezdívka: Marty, CR7</p>
+                </div>
+                <div class="float-box">
+                    <h3>Michal Špinar</h3>
+                    <p>Výška: 180 cm</p>
+                    <p>Číslo dresu: 6</p>
+                    <p>Přezdívka: Sněží</p>
+                </div>
             </div>
         </div>
     </div>
 
     <div class="clear"></div>
 
+    <div class="dont-print">
+        <div class="wrapper">
+            <h2>Ukázka sestavy</h2>
+            <div class="container">
+                <img src="pics/sestava.png" alt="základní sestava" class="imgcenter">
```

Comment:
pokud vim tak nas predmet se nevenuje MS Malovani.
myslel jsem trochu neco jineho, jak mas ty krabicky tak tam pridat jejich fotky a napravo od nich budou jejich informace

Reply:
až bude takový předmět akreditován, tak budu mít už aspoň natrénováno... jinak jsem tam tohle, co řikáš dodělal, hned to vypadá o něco líp